### PR TITLE
Story/quad 226

### PR DESCRIPTION
**Pull request by snilapwa**

### CODE REVIEW

File: Quadriga/src/main/webapp/WEB-INF/views/workspace/workspacedetails.jsp [hunk 1]
```diff
@@ -461,14 +467,16 @@ cursor: pointer;
...
                             textFileAccess = $("#" + textFileId).data("textaccess");
                             
-                            if (textFileAccess === "PUBLIC") {
-                                changedTextFileAccess = "Private";
-                            } else if (textFileAccess === "PRIVATE") {
-                                changedTextFileAccess = "Public";
-                            }
+                            var textFileAccessMap = new Object();
+                            textFileAccessMap["PUBLIC"] = "Are you sure you want to make this text file Private";
+                            textFileAccessMap["PRIVATE"] = "Are you sure you want to make this text file Public";
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/workspace/workspacedetails.jsp]: Question marks are missing for both questions.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by snilapwa** [comment]
Corrected file access change confirmation message, please help to review

File: Quadriga/src/main/webapp/WEB-INF/views/workspace/workspacedetails.jsp [hunk 2]
```diff
@@ -468,8 +468,8 @@ cursor: pointer;
                             textFileAccess = $("#" + textFileId).data("textaccess");
                             
                             var textFileAccessMap = new Object();
-                            textFileAccessMap["PUBLIC"] = "Are you sure you want to make this text file Private";
-                            textFileAccessMap["PRIVATE"] = "Are you sure you want to make this text file Public";
+                            textFileAccessMap["PUBLIC"] = "Are you sure you want to make this text file Private ?";
+                            textFileAccessMap["PRIVATE"] = "Are you sure you want to make this text file Public ?";
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/workspace/workspacedetails.jsp]: There is no blank before a period, question mark, or exclamation mark in a sentence.

**Comment by snilapwa** [comment]
Corrected message.

**Comment by jdamerow** [comment]
ok to test

**Review by jdamerow — APPROVED** [review]

**Pull request merged**